joomla-projects/privacy-framework
# [Feature] Incorporate Logging in Request Workflow

Pull Request for Issues #71 & #73

### Summary of Changes

- Removes `com_privacy` from the `com_actionlogs` config - the log plugins were duplicating log messages when the save events were triggered, and we really don't want this logging action to be user configurable
- Adds logs for all request related activities (created from both sides of the app, user confirms, and admin completes or invalidates the request)
- Adds a method to the `ActionlogsModelActionlogs` class to query all logs for an item
- Displays a request's log on the request item view

### Testing Instructions

With the patch applied, each step in the request workflow should add a log entry to the action logs table.  This is confirmable by viewing the action logs component or the request's info view.
